{"filter":false,"title":"bb8_move_in_square_service_server.py","tooltip":"/unit_3_services/src/bb8_move_in_square_service_server.py","undoManager":{"mark":-1,"position":-1,"stack":[]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":0,"column":0},"end":{"row":0,"column":0},"isBackwards":true},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1538112435951,"hash":"96999f2d6bf996211dfaed4d7f006dfbc4d3d535"}